openwisp_radius/api/freeradius_views.py

```diff
@@ -173,7 +174,11 @@ def get_user(self, request, username):
         return active user or ``None``
         """
         try:
-            user = auth_backend.get_users(username).filter(is_active=True)[0]
+            filter_kwargs = dict(is_active=True)
+            organization = Organization.objects.get(pk=request._auth)
+            if self._needs_identity_verification(organization):
+                filter_kwargs['registereduser__is_verified'] = True
```

Comment:
I'm not convinced of the implementation, because it will generate multiple queries, at least 2 (one to get the org and 1 to get the org radius settings).

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/ef3686e4dbe8c5e85a5bec1769da36eaafd87ea8)